cfy_manager/utils/node.py

```diff
@@ -35,7 +33,7 @@
 
 
 def get_node_id():
-    if not path.exists(STATUS_REPORTER_CONFIGURATION_PATH):
+    if not is_file(STATUS_REPORTER_CONFIGURATION_PATH):
```

Comment:
This check is now more correct, as the status reporter config path is the actual yaml file.